pyproject.toml

```diff
@@ -89,13 +89,13 @@ semantic_version = ">=2.9.0"
 sentry-sdk = "^1.5.6"
 simplejson = "^3.17.0"
 SPARQLWrapper = "^1.8.5"
-SQLAlchemy = "1.3.16"  # Pinned because >=1.3.17 is broken for us (circular constraints prevent deletes)
+SQLAlchemy = "1.4.41"  # pinned for stability
```

Comment:
say why that version? is that the latest as of today or something we've already tested or something else?